components/infobox/wikis/pubg/infobox_league_custom.lua

```diff
@@ -0,0 +1,224 @@
+---
+-- @Liquipedia
+-- wiki=pubg
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local League = require('Module:Infobox/League')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+local Tier = require('Module:Tier')
+local Template = require('Module:Template')
+local Class = require('Module:Class')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local PrizePoolCurrency = require('Module:Prize pool currency')
+
+local CustomLeague = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _args
+local _game
+
+local _TODAY = os.date('%Y-%m-%d', os.time())
+local _GAME = mw.loadData('Module:GameVersion')
+
+local _MODES = {
+	solo = 'Solos[[Category:Solos Mode Tournaments]]',
+	duo = 'Duos[[Category:Duos Mode Tournaments]]',
+	squad = 'Squads[[Category:Squads Mode Tournaments]]',
+	['2v2'] = '2v2 TDM[[Category:2v2 TDM Tournaments]]',
+	['4v4'] = '4v4 TDM[[Category:4v4 TDM Tournaments]]',
+	['war mode'] = 'War Mode',
```

Comment:
No category for `war mode`?

Reply:
the old template doesn't have a category set for that mode

Reply:
will added in next update to provide consistency, this is probably left in the dust on the old one back then